sonarr4kinstall.sh

```diff
@@ -63,16 +72,28 @@ fi
 
 echo_progress_start "Generating configuration"
 
+# Start sonarr to config
+systemctl stop sonarr.service >>$log 2>&1
```

Comment:
Why we stopping regular sonarr? No need?

Reply:
couldve just been an isolateed issue on my end. was having a hard time getting the config going and the 2nd instance up. if no one else has the issue it can be removed.